### DISCUSSION TITLE
需要保持的几个好习惯 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#27] [General]
# 需要保持的几个好习惯 | 倾旋的博客

在这里我梳理一些工作和生活需要保持的几个好习惯，用于时刻提醒自己继续保持。

https://payloads.online/archivers/2023/10/31/5e64d697-f9ae-49fa-aa14-006418b25325

**Comment by yangzhao917** [comment]
is Nice

**Comment by xc3z** [comment]
请问zsh-completions你是如何配置的，手动安装的吗？
我从extra仓库安装，发现安装完的文件并不在/usr/share/zsh/plugins/zsh-completions,而是在/usr/share/zsh/site-functions 里

**Reply by Rvn0xsy** [reply]
zsh有插件目录，存放在：`$ZSH/plugins/`

你可以直接将插件的仓库git clone到插件目录，然后在`~/.zshrc` 文件中启用插件：

```
 68 # Which plugins would you like to load?
 69 # Standard plugins can be found in $ZSH/plugins/
 70 # Custom plugins may be added to $ZSH_CUSTOM/plugins/
 71 # Example format: plugins=(rails git textmate ruby lighthouse)
 72 # Add wisely, as too many plugins slow down shell startup.
 73 plugins=(git docker zsh-syntax-highlighting zsh-autosuggestions zsh-completions copyfil    e emoji extract sudo)
```

**Comment by xc3z** [comment]
> zsh有插件目录，存放在：`$ZSH/plugins/`
> 
> 你可以直接将插件的仓库git clone到插件目录，然后在`~/.zshrc` 文件中启用插件：
> 
> ```
>  68 # Which plugins would you like to load?
>  69 # Standard plugins can be found in $ZSH/plugins/
>  70 # Custom plugins may be added to $ZSH_CUSTOM/plugins/
>  71 # Example format: plugins=(rails git textmate ruby lighthouse)
>  72 # Add wisely, as too many plugins slow down shell startup.
>  73 plugins=(git docker zsh-syntax-highlighting zsh-autosuggestions zsh-completions copyfil    e emoji extract sudo)
> ```
👌好的
